client/state/signup/segments/actions.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { SIGNUP_SEGMENTS_REQUEST, SIGNUP_SEGMENTS_SET } from 'state/action-types';
+
+import 'state/data-layer/wpcom/signup/segments';
```

Comment:
I just want to check, we import this file to trigger the calling of `registerHandlers( ... )` right?

Reply:
Just so we can bundle them when we need them. See: https://github.com/Automattic/wp-calypso/commit/e80cda04df36df1cc0bf7b7d2716920d5db00948

Reply:
Nice! Thanks @ramonjd :)